source/credentials_provider_default_chain.c

```diff
@@ -279,20 +282,36 @@ struct aws_credentials_provider *aws_credentials_provider_new_chain_default(
     if (environment_provider == NULL) {
         goto on_error;
     }
-
     providers[index++] = environment_provider;
 
-    struct aws_credentials_provider_profile_options profile_options;
-    AWS_ZERO_STRUCT(profile_options);
-    profile_options.bootstrap = options->bootstrap;
-    profile_options.shutdown_options = sub_provider_shutdown_options;
+    struct aws_credentials_provider_sts_web_identity_options sts_web_identity_options = {
+        .shutdown_options = sub_provider_shutdown_options, .bootstrap = options->bootstrap};
+
+    sts_web_identity_provider = aws_credentials_provider_new_sts_web_identity(allocator, &sts_web_identity_options);
+    if (sts_web_identity_provider != NULL) {
+        providers[index++] = sts_web_identity_provider;
+        aws_atomic_fetch_add(&impl->shutdowns_remaining, 1);
+    }
+
+    struct aws_credentials_provider_profile_options profile_options = {
+        .shutdown_options = sub_provider_shutdown_options, .bootstrap = options->bootstrap};
     profile_provider = aws_credentials_provider_new_profile(allocator, &profile_options);
     if (profile_provider != NULL) {
         providers[index++] = profile_provider;
         /* 1 shutdown call from the profile provider's shutdown */
         aws_atomic_fetch_add(&impl->shutdowns_remaining, 1);
     }
 
+    struct aws_credentials_provider_process_options process_options = {.shutdown_options =
```

Comment:
This disgusting format is done by clang-format

Reply:
comma after sub_provider_shutdown_options